### ISSUE TITLE
difference between Custom text facet .... and Custom Numeric Facet ...

### ISSUE THREAD

**Issue by magdmartin** [OpenRefine/OpenRefine#990]
Is there a difference between the two? should we just keep one for usability purpose?

**Comment by tfmorris** [comment]
Please use the mailing lists for questions.

Text facets and numeric facets are handled completely differently.  One creates a list of choices and the other shows a histogram of counts vs numerical values.

**Comment by magdmartin** [comment]
I am talking about the _custom__text facet_ and _custom numeric facet_ 
choices number 5 and 6 in the facet sub menu.
Both options open the same transformation window.

**Comment by tfmorris** [comment]
I think we are talking past each other.

A custom TEXT facet and a custom NUMERIC facet are, at the end of the day, just a text facet and numeric facet as I stated in my original reply.  Yes, they both open the expression preview window -- that's the "custom" part, but what they expect as a resulting value and what they do with it afterwards are completely different.

Why are we discussing this?  Why aren't we focused on getting 2.6 tested and released?  What part of "use the mailing lists" for things which aren't bugs or enhancements was unclear?
